index.js

```diff
@@ -94,20 +95,20 @@ app.use((req, res, next) => {
 
 // middleware function to check for logged-in users
 var sessionChecker = (req, res, next) => {
-    if (req.session.user && req.cookies.user_sid) {
-        res.redirect('/dashboard');
+    if (!req.session.user && !req.cookies.user_sid) {
+        res.redirect('/login')
     }
     else {
-        next();
+        next()
     }
 };
 
 app.get('/', sessionChecker, (req, res) => {
-    res.redirect('/login');
+    res.redirect('/dashboard');
```

Comment:
updated this because with sessionChecker in place, if someone is logged in and tries to go to /, they should be redirected to the landing page vs login page